src/utils/parserUtils.ts

```diff
@@ -8,6 +8,18 @@ import { getMarkByName } from '../services/markService';
 import { updateMarks } from '../services/markService';
 import { getModelByName, saveNewModel } from '../services/modelService';
 
+export const transformOnlinerDate = (onlinerDate: string) => {
+  let date = onlinerDate.substring(0, 10);
```

Comment:
Please use moment.js here.

Reply:
Should we really add an external library for only one small piece of code?